examples/hub_demo/hub_demo.ino

```diff
@@ -293,7 +293,7 @@ unsigned int getstrdescr( unsigned int addr, uint8_t idx )
     Serial.print("Error retrieving LangID table ");
     return ( rcode );
   }
-  langid = word(buf[3], buf[2]);
+  langid = makeWord(buf[3], buf[2]);
```

Comment:
Sorry, but don't use the makeWord function either, as this might not be available on non-Arduino boards. Simply just do it yourself:
```C++
(buf[3] << 8) | buf[2];
```